docs/.vuepress/public/static/styles.css

```diff
@@ -128,7 +128,7 @@ body .live-preview pre {
   padding: 10px;
   font-size: 18px;
   color: #FFF;
-  background: darkcyan;
+  background: #008484;
```

Comment:
`darkcyan` is just slightly too bright to meet AA standards

Reply:
OK. I just picked it because I wanted to use a named color and this one looked nice, but it's a much better idea to have examples that pass a11y standards in case someone were to copy/paste.